u/index.md

```diff
@@ -0,0 +1,3 @@
+# List of artists starting with U:
+
+  Oops! It seems as if there isn't any Malagasy artist's name start with 'U'. 
```

Comment:
This looks good!
I'd suggest a rewording though:

```
Oops! It seems there aren't any Malagasy artists with names starting with 'U'.
```

Reply:
Could you check it now, please?

Reply:
How about #32, because #15 is already existed.